control-plane-operator/hostedclusterconfigoperator/operator/config.go

```diff
@@ -106,20 +106,21 @@ func Mgr(cfg, cpConfig *rest.Config, namespace string) ctrl.Manager {
 		},
 		NewCache: cache.BuilderWithOptions(cache.Options{
 			SelectorsByObject: cache.SelectorsByObject{
-				&corev1.Namespace{}:         allSelector,
-				&configv1.Infrastructure{}:  allSelector,
-				&configv1.DNS{}:             allSelector,
-				&configv1.Ingress{}:         allSelector,
-				&operatorv1.Network{}:       allSelector,
-				&configv1.Network{}:         allSelector,
-				&configv1.Proxy{}:           allSelector,
-				&configv1.Build{}:           allSelector,
-				&configv1.Image{}:           allSelector,
-				&configv1.Project{}:         allSelector,
-				&configv1.ClusterVersion{}:  allSelector,
-				&configv1.FeatureGate{}:     allSelector,
-				&configv1.ClusterOperator{}: allSelector,
-				&configv1.OperatorHub{}:     allSelector,
+				&corev1.Namespace{}:           allSelector,
+				&configv1.Infrastructure{}:    allSelector,
+				&configv1.DNS{}:               allSelector,
+				&configv1.Ingress{}:           allSelector,
+				&operatorv1.Network{}:         allSelector,
+				&configv1.Network{}:           allSelector,
+				&configv1.Proxy{}:             allSelector,
+				&configv1.Build{}:             allSelector,
+				&configv1.Image{}:             allSelector,
+				&configv1.Project{}:           allSelector,
+				&configv1.ClusterVersion{}:    allSelector,
+				&configv1.FeatureGate{}:       allSelector,
+				&configv1.ClusterOperator{}:   allSelector,
+				&configv1.OperatorHub{}:       allSelector,
+				&operatorv1.CloudCredential{}: allSelector,
```

Comment:
nit: We wouldn't need to include CloudCredential here given we are creating it it'll contain cacheLabelSelector already and so we'll catch only that resource?